src/order/filter/filter_product.lean

```diff
@@ -179,7 +174,7 @@ instance [comm_semiring β] : comm_semiring β* :=
 instance cr [comm_ring β] : comm_ring β* := 
 {..filter_product.r β φ, ..filter_product.cs β φ}
 
-instance zo [zero_ne_one_class β] {U : is_ultrafilter φ} : zero_ne_one_class β* :=
+instance zo [zero_ne_one_class β] (U : is_ultrafilter φ) : zero_ne_one_class β* :=
```

Comment:
I'm not sure about these abbreviated names. These names are hard to guess with autocompletion. Usually the instances get a good name by default... If you get a naming conflict that is an indication you should be in some namespace.
Idem for `cr` a couple lines up, and `dr` a couple lines down.